### DISCUSSION TITLE
Is the loading project screen supposed to change in color?

### DISCUSSION THREAD

**Discussion by medlans** [ScratchAddons/ScratchAddons#6384] [Help]
I have Scratch 2.0 editor colors, but it's still blue, and I thought it was supposed to change to grey (at least, that's what it did before):
https://scratch.mit.edu/discuss/youtube/uz9BjSqF7_w/

**Comment by mxmou** [comment]
It used to be affected by the highlight color setting, but it was changed to always be blue so that it looks the same as in vanilla Scratch with default settings. A separate setting for the loading screen background might be added in the future.
